### PR TITLE
Update SSO guides for self serve

### PR DESCRIPTION
**Pull request by harrietgrace**
Still a draft, will add to it over the day 👍🏻

### CODE REVIEW

**Comment by harrietgrace** [comment]
@plasticine check out the Okta page in the preview and let me know what your feels are! https://buildkite-docs-review-pr-497.herokuapp.com/docs/integrations/sso/okta Steps 2-4 are different, and step 1 stayed the same.  

I was thinking of putting in a second screenshot of the actual Okta page, and maybe making the first screenshot smaller or more specific or something... it seems very large and whole-page-y at the moment 🤷🏻‍♀️

**Comment by plasticine** [comment]
This is awesome, @harrietgrace! 👏 The note on disabling is so good. Such an awesome thing to have an explicit note one.

> Add Buildkite in Okta

Is there another way to word this, or is “Add XYZ in Okta” the way they refer to their app ecosystem? I dunno if I’m thinking wrong (I must admit I’m not an Okta poweruser), but this jars a little bit for me. 

Is there a way we can say something like “Add the Buildkite App to your Okta Account”, or something?

File: pages/integrations/sso.md.erb [hunk 1]
```diff
@@ -47,6 +47,18 @@ For more information on the SAML user attributes Buildkite supports, see the [Cu
...
+If you need to edit your SSO settings, temporarily stop logins using SSO, or want to delete your SSO provider, you'll first need to disable it.
+
+There are two ways to disable a provider:
+- using the 'disable' button in your SSO provider Settings
+- using GraphQL
+
+After you've disabled your provider you can edit, delete, or re-enable it.
+
+While disabled, logins will no longer go through your SSO provider. Users accessing your organisation data will be required to log in using a username and password. If users don't have a password for their Buildkite account and need access while SSO is disabled, they can perform a 'forgotten password' reset. For assitance with this process, contact your organization admin or Buildkite support.
+
```
Comment (plasticine) [inline pages/integrations/sso.md.erb]: Suuuuuuuuuuper good.

File: pages/integrations/sso.md.erb [hunk 2]
```diff
@@ -47,6 +47,18 @@ For more information on the SAML user attributes Buildkite supports, see the [Cu
 
 If you'd prefer a self-service setup, you can complete the Buildkite side yourself using the Buildkite GraphQL explorer. Check out our guide to [Setting up SSO with GraphQL](/docs/integrations/sso/sso-setup-with-graphql).
 
+## How do I disable SSO?
+
+If you need to edit your SSO settings, temporarily stop logins using SSO, or want to delete your SSO provider, you'll first need to disable it.
+
+There are two ways to disable a provider:
+- using the 'disable' button in your SSO provider Settings
+- using GraphQL
```
Comment (plasticine) [inline pages/integrations/sso.md.erb]: Do we usually add references or examples for graphql things?

Comment (harrietgrace) [reply pages/integrations/sso.md.erb]: Yeah, I'll link to the graphql sso guide, it's got example code 👍🏻

Comment (plasticine) [reply pages/integrations/sso.md.erb]: Ahhh, of course! Bueno 👌

**Comment by harrietgrace** [comment]
Alrighty, @plasticine I'm going to copy the new bits into the other guides and sort out the provider-specific screenshots. Then it should be ready to go 🚀

**Comment by harrietgrace** [comment]
Okay! So these have been updated to be completely self-serve:

- Okta
- OneLogin
- Google G Suite

These have been updated to be mostly self-serve (the still require people to get a URL from BK support before the start their setup - they'll need it before they get their x509 cert and stuff):

- ADFS
- Custom SAML

And Google G Cloud hasn't been updated at all because we don't have a button for it in the app yet 👍🏻

**Comment by harrietgrace** [comment]
@plasticine any last minute changes or additions? I revised some of the bits on the overview page and checked through the rest of the guides. I'm happy to merge when you are 👍🏻

File: pages/integrations/sso.md.erb [hunk 3]
```diff
@@ -24,28 +24,39 @@ Team maintainers can still invite users to their Buildkite organisation, and are
...
+You will need to set up your SSO identity provider yourself. There are guides available for several of the commonly used providers. See the provider-specific sections below for further information.
 
 ### Using Google G Suite or Github
 
-Google and Github both use OAuth rather than SAML. See the [Google G Suite setup guide](/docs/integrations/sso/g-suite) for a list of the information you'll need to give Buildkite to get started.
+Google and Github both use OAuth rather than SAML. See the [Google G Suite setup guide](/docs/integrations/sso/g-suite) for a list of the information you'll need to get started.
 
 You can also set up OAuth providers with GraphQL, see our [GraphQL SSO guide](/docs/integrations/sso/sso-setup-with-graphql) for steps and code samples.
 
 ### Using Okta, OneLogin, AD FS, or a custom SAML Provider
```
Comment (plasticine) [inline pages/integrations/sso.md.erb]: `AD FS`?

File: pages/integrations/sso.md.erb [hunk 4]
```diff
@@ -24,28 +24,39 @@ Team maintainers can still invite users to their Buildkite organisation, and are
...
 
 There are guides available for setting up Buildkite with the following SAML identity providers: [Okta](/docs/integrations/sso/okta), [OneLogin](/docs/integrations/sso/onelogin), [Google Cloud Identity](/docs/integrations/sso/g-cloud-identity), [AD FS](/docs/integrations/sso/adfs), and a general guide for [custom identity providers](/docs/integrations/sso/custom-saml).
```
Comment (plasticine) [inline pages/integrations/sso.md.erb]: `AD FS`

File: pages/integrations/sso.md.erb (hunk 3 above)
Comment (plasticine) [reply pages/integrations/sso.md.erb]:
```suggestion
 ### Using Okta, OneLogin, ADFS, or a custom SAML Provider
```

File: pages/integrations/sso.md.erb (hunk 4 above)
Comment (plasticine) [reply pages/integrations/sso.md.erb]:
```suggestion
 There are guides available for setting up Buildkite with the following SAML identity providers: [Okta](/docs/integrations/sso/okta), [OneLogin](/docs/integrations/sso/onelogin), [Google Cloud Identity](/docs/integrations/sso/g-cloud-identity), [ADFS](/docs/integrations/sso/adfs), and a general guide for [custom identity providers](/docs/integrations/sso/custom-saml).
```

**Comment by harrietgrace** [comment]
Thanks! I was all for keeping the space in AD FS, but Tim thought no space would be better for seo and such things 😄

**Pull request merged**